src/module/classes/GmScreenApplication.ts

```diff
@@ -151,6 +151,7 @@ export class GmScreenApplication extends Application {
    */
   toggleGmScreenVisibility(expanded: boolean = !this.expanded) {
     this.expanded = expanded;
+    Hooks.callAll("gmScreen", this, expanded);
```

Comment:
Use `MODULE_ID` here as well. Also the more I think about this, the more I'd like the last arg provided to be an object, that data model will be more extensible in the future if other things should be added to this hook.

```
Hooks.callAll(MODULE_ID, this, { expanded });
```